### ISSUE TITLE
LaTex Math Support

### ISSUE THREAD

**Issue by ghost** [n457/Uncolored#2]
@n457 , I **love** Uncolored. Love the interface, and love the simplicity with _power_. However, I do a lot of math, and would love LaTex math support. Is this coming down the pipeline?

**Comment by n457** [comment]
Hi @codesharkasm, thanks :)

Adding LaTex Math support was not planned at start. I did a little research and **it seems pretty complicated to set up**.

I found :
- [MathJax](https://www.mathjax.org/) which outputs **classed HTML tags with inline CSS styles**, **this is pretty ugly** in terms of code, and I absolutly want to keep a document HTML code **clean (without to much _class_ attributes & no _style_ attributes at all)**.
- [Textogif](http://www.fourmilab.ch/webtools/textogif/) **has too many prerequisites**.

The only **light & simple** solution I see is to **convert your math formula into image** via one of the several online services like [latex2png](http://latex2png.com/) or [TeX2PNG](http://frog.isima.fr/bruno/share/tex2png/) and import the local (or remote) image into the document (importing local images **and incorporate them** in a data64 format in a document will be possible in a near future).

What do you think ?

**Comment by ghost** [comment]
@n457 
True. I see what you mean. I checked the code of a sample MathJax exported document. It was horrible.

But before you write this one off, download and install [Typora](http://typora.io) and check out [Classeur](http://classeur.io). Typora isn't really that nice looking compared to (Un)colored, but has a few more features such as math, TOC, tables, and **auto-closing syntax!** (for example, typing `(` will bring up `()` and will put the caret between the two. Same with `"`, `[`, `**`, etc.) Check out the code and stuff for Typora (it's built on electron). Classeur is both nice and powerful, but is not native. 

If Mathjax doesn't work, is it possible to integrate TeX2PNG into the editor itself so that when I type some MathJax code into (Un)colored, it translates it into a png? Sorry, I'm a bit of a novice in development. I'm a student and I use markdown programs a lot. I'm still on the hunt for the _perfect_ MDown editor.
